tensorflow/core/kernels/mkl/mkl_quantized_conv_ops_test.cc

```diff
@@ -296,29 +309,33 @@ TEST_F(QuantizedConv2DTest, SmallS8) {
   const float image_min = -127.0f;
   const float image_max = 127.0f;
 
-  TF_ASSERT_OK(NodeDefBuilder("quantized_conv_op", "_MklQuantizedConv2D")
-                   .Input(FakeInput(DT_QINT8))  // Input
-                   .Input(FakeInput(DT_QINT8))  // Filter
-                   .Input(FakeInput(DT_FLOAT))  // Min input
-                   .Input(FakeInput(DT_FLOAT))  // Max input
-                   .Input(FakeInput(DT_FLOAT))  // Min filter
-                   .Input(FakeInput(DT_FLOAT))  // Max filter
-                   //  MKL metadata tensors //
-                   .Input(FakeInput(DT_UINT8))
-                   .Input(FakeInput(DT_UINT8))
-                   .Input(FakeInput(DT_UINT8))
-                   .Input(FakeInput(DT_UINT8))
-                   .Input(FakeInput(DT_UINT8))
-                   .Input(FakeInput(DT_UINT8))
-                   ///////////////////////////
-                   .Attr("Tinput", DataTypeToEnum<qint8>::v())
-                   .Attr("Tfilter", DataTypeToEnum<qint8>::v())
-                   .Attr("T", DataTypeToEnum<quint8>::v())
-                   .Attr("padding", "VALID")
-                   .Attr("out_type", DataTypeToEnum<qint32>::v())
-                   .Attr("strides", {1, stride, stride, 1})
-                   .Attr("_kernel", "QuantizedMklOp")
-                   .Finalize(node_def()));
+  NodeDefBuilder builder =
+      NodeDefBuilder("quantized_conv_op", NativeFormatEnabled()
+                                              ? "_MklNativeQuantizedConv2D"
+                                              : "_MklQuantizedConv2D")
+          .Input(FakeInput(DT_QINT8))  // Input
+          .Input(FakeInput(DT_QINT8))  // Filter
+          .Input(FakeInput(DT_FLOAT))  // Min input
+          .Input(FakeInput(DT_FLOAT))  // Max input
+          .Input(FakeInput(DT_FLOAT))  // Min filter
+          .Input(FakeInput(DT_FLOAT))  // Max filter
+          .Attr("Tinput", DataTypeToEnum<qint8>::v())
+          .Attr("Tfilter", DataTypeToEnum<qint8>::v())
+          .Attr("T", DataTypeToEnum<quint8>::v())
+          .Attr("padding", "VALID")
+          .Attr("out_type", DataTypeToEnum<qint32>::v())
+          .Attr("strides", {1, stride, stride, 1})
+          .Attr("_kernel", "QuantizedMklOp");
```

Comment:
Should `T` and `_kernel` be in `if (!NativeFormatEnabled)` instead?

Reply:
For `T` yes. For `_kernel` no, per the MKL rewrite algorithm, label is required if the op name changes (i.e. adding _MklNative).